NYC_Summit18_Developer_Workshop/IDCT_Lab.md

```diff
@@ -19,9 +19,9 @@ This tutorial is designed to teach the fundamentals of the SDAccel development e
 
 The kernel used in this tutorial is an Inverse Discrete Cosine Transform (IDCT), a function widely used in audio/image codecs such as HEVC. 
 
-To simplify this tutorial, a workspace preloaded with the initial project configuration is provided in the workshop repository.
+To simplify this tutorial, a workspace preloaded with the initial project configuration is provided in the workshop directory within the repository.
 
-Please also note that although the entire tutorial is performed on an F1 instance, only the final step of this tutorial really needs to be run on F1. All the interactive development, profiling and optimization steps would normally be performed on-premise or on a cost-effective AWS EC2 instance such as C4. However, to avoid switching from C4 to F1 instances during this workshop, all the steps are performed on the F1 instance.
+Please also note that although the entire tutorial is performed on an F1 instance, only the final step of this tutorial really needs to be run on F1. All the interactive development, profiling and optimization steps would normally be performed on-premise or on a cost-effective AWS EC2 instance such as C5 or Z1d. However, to avoid switching from other instance to F1 instances during this workshop, all the steps are performed on the F1 instance.
```

Comment:
```suggestion
Please also note that although the entire tutorial is performed on an F1 instance, only the final step of this tutorial really needs to be run on F1. All the interactive development, profiling and optimization steps would normally be performed on-premise or on a cost-effective AWS EC2 instance such as C5 or Z1d. However, to avoid switching from other instances to an F1 instance during this workshop, all the steps are performed on an F1 instance.
```